FreeRTOS-Plus/Demo/corePKCS11_Windows_Simulator/main.c

```diff
@@ -68,6 +73,7 @@ static void prvPKCS11DemoTask( void * pvParameters )
     #endif
     configPRINTF( ( "---------Finished DEMO---------\r\n" ) );
 
+    vPlatformStopLoggingThreadAndFlush();
```

Comment:
Not required in this PR, but we should add this in fault handlers as well.

Reply:
Call `vPlatformStopLoggingThreadAndFlush()` in `vAssertCalled()`. `vPlatformStopLoggingThreadAndFlush()` does nothing if the logging is not initialized before.